specs/phase1/beacon-chain.md

```diff
@@ -327,6 +340,8 @@ class BeaconState(Container):
     # at RANDAO reveal period % EARLY_DERIVED_SECRET_PENALTY_MAX_FUTURE_EPOCHS
     exposed_derived_secrets: Vector[List[ValidatorIndex, MAX_EARLY_DERIVED_SECRET_REVEALS * SLOTS_PER_EPOCH],
                                     EARLY_DERIVED_SECRET_PENALTY_MAX_FUTURE_EPOCHS]
+    custody_chunk_challenge_records: List[CustodyChunkChallengeRecord, MAX_CUSTODY_CHUNK_CHALLENGE_RECORDS]
```

Comment:
The problem is that we have to use this one, as we aren't doing "object diffs" anymore and so the full definition of the beacon chain state has to be in the beacon chain spec. Thus, this constant is part of the beacon spec.

Reply:
Yeah but so are `MAX_CUSTODY_CHUNK_CHALLENGES`, `MAX_CUSTODY_CHUNK_CHALLENGE_RESPONSES`, `MAX_CUSTODY_KEY_REVEALS`, `MAX_EARLY_DERIVED_SECRET_REVEALS`, and `MAX_CUSTODY_SLASHINGS`.  The problem is that `beacon-chain.md` and `custody-game.md` are depending on each other now, although these constants definitions are reordered in `setup.py`.

It's not perfect but at least it would be better if we can put these similar constants to the same table now. Either in `beacon-chain.md` or `custody-game.md` is fine by me :)

Reply:
OK, didn't know that. I will add it do the custody game spec then.